### PR TITLE
Implementation of sk_sk translation ver2

### PR DESCRIPTION
**Pull request by pepo939393**
Describe the overall purpose of your pull request in a few sentences, keeping
things at a high level. If this request is in response to a particular issue,
include a reference to it here.

Make sure that your pull request follows the guidelines specified in the
[Guidelines for Contributing](../docs/CONTRIBUTING.md).

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -146,6 +146,9 @@ elseif(LOCALE MATCHES "^de[_-]")
 elseif(LOCALE MATCHES "^sv[_-]")
   set(STUMPLESS_LANGUAGE "sv-SE")
   set(USE_LOCALE_SV_SE TRUE)
+elseif(LOCALE MATCHES "^sk[_-]")
+  set(STUMPLESS_LANGUAGE "sk-SK")
+  set(USE_LOCALE_SK_ SK TRUE)
```
Comment (goatshriek) [inline CMakeLists.txt]: It looks like there is an accidental space between `SK_` and `SK` that will prevent this `set` command from functioning as intended.

File: CMakeLists.txt [hunk 2]
```diff
@@ -176,7 +179,6 @@ set(STUMPLESS_SOURCES
   ${PROJECT_SOURCE_DIR}/src/validate.c
 )
 
-
```
Comment (goatshriek) [inline CMakeLists.txt]: leave this whitespace in (it's unrelated to your change)

File: CMakeLists.txt [hunk 3]
```diff
@@ -443,8 +445,6 @@ else()
     ${PROJECT_SOURCE_DIR}/docs/examples/socket/socket_example.c
   )
 endif(NOT ENABLE_SOCKET_TARGETS)
-
-
```
Comment (goatshriek) [inline CMakeLists.txt:447]: leave this whitespace in as well (it's unrelated to your change)

File: CMakeLists.txt [hunk 4]
```diff
@@ -462,6 +462,8 @@ if(NOT STUMPLESS_WINDOWS_EVENT_LOG_TARGETS_SUPPORTED)
     SOURCES ${PROJECT_SOURCE_DIR}/test/function/config/wel_unsupported.cpp
   )
 else()
+  set(STUMPLESS_WINDOWS_EVENT_LOG_TARGETS_SUPPORTED TRUE)
+
```
Comment (goatshriek) [inline CMakeLists.txt]: you can remove these lines - they were changed in a recent commit (only a few days ago), so I suspect they are just left in your branch from the older version

File: include/private/config/locale/wrapper.h [hunk 5]
```diff
@@ -29,6 +29,8 @@
 #	 include "private/config/locale/de-de.h"
 #  elif defined USE_LOCALE_SV_SE
 #    include "private/config/locale/sv-se.h"
+#  elif defined USE_LOCALE_SK_SK
+#   include "private/config/locale/sk-sk.h"
```
Comment (goatshriek) [inline include/private/config/locale/wrapper.h]: slight indentation mismatch here; you just need one more space to bring it into line with everything else

**Review by goatshriek — CHANGES_REQUESTED** [review]
These changes look good! I have annotated some whitespace fixes as well as a leftover line from an older commit that should be removed.

In addition to the above comments, please update the `.github/workflows/build.yml` file to include a CI build for this locale as well. This will help catch things like the space in the set command in the `CMakeLists.txt` file, as well as any future regressions that affect this locale.

And one last update: be sure to add `:slovakia:` to the README.md file (line 28) so that others can see the hard work you've done!

**Comment by pepo939393** [comment]
Hi! I repare the stuff I should. I hope it is fine now.

**Comment by goatshriek** [comment]
The fixes look good! Please update the `.github/workflows/build.yml` file with a new CI build for this locale, and after that I will merge the pull request. There is a little more detail on how to do this in the [localization documentation](https://github.com/goatshriek/stumpless/blob/latest/docs/localization.md#defining-a-new-locale).

**Comment by pepo939393** [comment]
I upadated it.

**Comment by goatshriek** [comment]
Awesome! My apologies, I was a little impatient and should have waited a bit longer before responding.

Thanks again for this contribution!

**Comment by pepo939393** [comment]
Thank you for contribution too!

**Pull request merged**